packages/docs/src/routes/docs/(qwik)/components/state/index.mdx

```diff
@@ -22,24 +22,23 @@ contributors:
   - gkatsanos
   - adamdbradley
   - mhevery
+  - un33k
 ---
 
 import CodeSandbox from '../../../../../components/code-sandbox/index.tsx';
 
 # State
 
-State management is an important part of any app. In Qwik, we can differentiate between two types of state, reactive and static:
+Reactive state management is an approach to managing the state of an application in response to changes in data or user interactions.
 
-1. Static state is anything that can be serialized: a string, number, object, array... anything.
-2. A reactive state on the other hand is created with `useSignal()` or `useStore()`.
+In Qwik:
 
-It is important to notice that state in Qwik is not necessarily a local component state, but rather an app state that can be instantiated by any component.
+  - Reactive state is created using the `useSignal()` or `useStore()` functions, (aka hooks).
+  - Reactive states can be shared with other components across the application to update them in response to changes.
 
 ## `useSignal()`
 
-Use `useSignal()` to create a reactive signal (a form of state). The `useSignal()` takes an initial value and returns a reactive signal.
-
-The reactive signal returned by `useSignal()` consists of an object with a single property `.value`. If you change the `value` property of the signal, any component that depends on it will be updated automatically.
+The `useSignal()` hook accepts an `initial value`, creates, and returns a `reactive state`, also known as a `signal`. This signal is an object with a single property, `.value`. Multiple components can consume the signal. When any component with access to the signal modifies the `.value` property, all dependent components will automatically update in response to the change.
```

Comment:
wondering if we can split this paragram into two, seems like it's talking about several concepts! we are finding that keeping paragraphs focus on a single concept can boost GPT results

Reply:
Yeah, I agree, the split look much better and to-the-point.